tasks/pkg-redhat/install-apm-inject-latest.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Install the APM inject library
```

Comment:
As noted in the APT installation file, let's try to collapse the installation tasks into a single one to save execution time.